tests/x11/network/hwsim_wpa2_enterprise_setup.pm

```diff
@@ -81,6 +82,8 @@ sub adopt_apparmor {
 
 sub reload_services {
     enter_cmd 'echo "# reload required services"';
+    # Need stop wicked first, then restart NetworkManager
+    systemctl 'stop wicked' if (is_sle);
```

Comment:
I would suggest `systemctl 'stop wicked' if (script_run('systemctl is active wicked') ==0);`